hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/HtsSystemContract.java

```diff
@@ -106,11 +108,18 @@ private static FullResult resultOfExecuting(
 
                 if (responseCode == SUCCESS) {
                     final var output = pricedResult.fullResult().result().getOutput();
+                    var updater = (ProxyWorldUpdater) frame.getWorldUpdater();
+                    final var senderId = ((ProxyEvmAccount) updater.getAccount(frame.getSenderAddress())).hederaId();
+
                     enhancement
                             .systemOperations()
                             .externalizeResult(
                                     contractFunctionResultSuccessFor(
-                                            pricedResult.fullResult().gasRequirement(), output, HTS_CONTRACT_ID),
+                                            pricedResult.fullResult().gasRequirement(),
```

Comment:
Would this be externalizing a `UtilPrngTransactionBody` for all view calls?

Reply:
With the new changes I made - it will not 😄